### DISCUSSION TITLE
Cursor visibility

### DISCUSSION THREAD

**Discussion by gitophile** [SlRvb/Obsidian--ITS-Theme#11] [Q&A]
I love the theme.  Not using any CSS yet cant see the cursor on EDIT mode.  Is that by design??

Greg

**Comment by gitophile** [comment]
Moved by author to forum.....

**Comment by SlRvb** [comment]
Ah sorry bout the late response 😥 but I'm not too sure what's going wrong with the cursor and colors, I can't quite replicate those errors on my end. Could you show me a screenshot or gif of what's happening for you?
